### DISCUSSION TITLE
TextPanel: Padding hides content if panel is resized to one text row

### DISCUSSION THREAD

**Discussion by peterholmberg** [grafana/grafana#38220] [Dashboards, Panels & Visualizations]
**What happened**:
If you in a text panel have a single row of text and you resize the panel to only accommodate that line. The panel will show a scrollbar. When you hover over the panel, the panel menu will also over the panel content.

Scrollbar to the right
![Screenshot from 2021-02-12 14-11-58](https://user-images.githubusercontent.com/946275/107772254-4ea78980-6d3c-11eb-9d8a-1fbcf78666cb.png)

Panel menu covering the panel
![Screenshot from 2021-02-12 14-13-23](https://user-images.githubusercontent.com/946275/107772413-81ea1880-6d3c-11eb-97ab-33328c964980.png)

**What you expected to happen**:
The panel should be resizable to only show the content without showing a scrollbar.

**How to reproduce it (as minimally and precisely as possible)**:
1. Create a text panel.
2. In edit, remove the `#Title` part and remove the Panel title.
3. Go back to the Dashboard and resize the panel to only show one text row.
4. There'll be a scrollbar if you hover the text and the pane and the panel title should cover the rest of the panel.

**Anything else we need to know?**:

**Environment**:
- Grafana version: 7.5 pre
- Data source type & version: n/a
- OS Grafana is installed on: Ubuntu 20.10
- User OS & Browser: Ubuntu 20.10 / latest Chrome
- Grafana plugins: n/a
- Others:

**Comment by torkelo** [comment]
I would label this a feature request, this not a supported use case (ie has never worked)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
